searx/autocomplete.py

```diff
@@ -36,7 +25,26 @@ def get(*args, **kwargs):
     return http_get(*args, **kwargs)
 
 
-def dbpedia(query, lang):
+def brave(query, lang):
+    # brave search autocompleter
+    url = 'https://search.brave.com/api/suggest?'
+    url += urlencode({'q': query})
+    country = 'all'
+    if lang in _brave:
+        country = lang
+    kwargs = {'cookies': {'country': country}}
+    resp = get(url, **kwargs)
+
+    results = []
+
+    if resp.ok:
+        data = loads(resp.text)
```

Comment:
`data = resp.json()` doesn't trigger the charset detection for each request (see the logs).

the `json()` method has shortcut to detect UTF encoding: https://github.com/encode/httpx/blob/321d4aa5097fe7f24cdfed7191c44de589294780/httpx/_models.py#L1514

Reply:
> see the logs ..

I debugged ..

```
(Pdb) l
 39  	    results = []
 40  	
 41  	    if resp.ok:
 42  	        import pdb
 43  	        pdb.set_trace()
 44  ->	        data = loads(resp.text)
 45  	        for item in data[1]:
 46  	            results.append(item)
 47  	    return results
 48  	
 49  	
(Pdb) resp.text
DEBUG   charset_normalizer            : override steps (5) and chunk_size (512) as content does not fit (153 byte(s) given) parameters.
INFO    charset_normalizer            : ascii passed initial chaos probing. Mean measured chaos is 0.000000 %
DEBUG   charset_normalizer            : ascii should target any language(s) of ['Latin Based']
INFO    charset_normalizer            : ascii is most likely the one. Stopping the process.
```

what means this logs are not related to json parser.  I assume the messages are related to a typecast of `resp.text` .. but I don't really know what I can do ...

Reply:
To avoid such issues from httpx.Response, we need to use httpx.Response.json() / fixed in e9588b70

@dalf: Should we replace `json.load(resp.text)` by `resp.json()` entirely? (I can do it in another PR)